### ISSUE TITLE
Creating virtualenvs in an infinite loop

### ISSUE THREAD

**Issue by zsol** [omnilib/thx#71]
### Description
While setting up thx for libcst, I noticed it keeps creating virtualenvs in an endless loop.

Repro:
add this to libcst's `pyproject.toml`:
```
[tool.thx]
python_versions = ["3.10"]

[tool.thx.jobs.lint]
run = [
    "./check_copyright.sh",
]
```

debug logs [here](https://gist.github.com/zsol/50e2e2d1e0f9779c776f55f030331168)

### Details

* OS: Windows
* Python version: 3.10.4
* thx version:0.5.1
* Can you repro on main? dunno
* Can you repro in a clean virtualenv? yes

**Comment by zsol** [comment]
Maybe related to #41 ?

**Comment by zsol** [comment]
Ah no, it's because watch mode restarts venv building because during venv building there are filesystem events coming in. `thx lint` works fine, `thx -w lint` is b0rken

**Comment by amyreese** [comment]
I don't have a Windows machine to test on, but I believe this is now fixed in v0.6.0.
